src/SDKs/_metadata/.txt

```diff
@@ -0,0 +1,17 @@
+﻿Executing AutoRest command
+cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/master/specification//readme.md --csharp --version=latest --reflect-api-versions --csharp-sdks-folder=F:\Git\Sync\azure-sdk-for-net\tools\..\src\SDKs\
+2018-09-04 21:06:14 UTC
+1) azure-rest-api-specs repository information
+GitHub fork: Azure
+Branch:      master
```

Comment:
Please delete this file and regenerate the code

Reply:
Done.  Deleted ".txt" and reran generate.ps1 which updated "datamigration_resource-manager.txt"